website/docs/r/security_group_rule.html.markdown

```diff
@@ -87,3 +87,23 @@ In addition to all arguments above, the following attributes are exported:
 * `to_port` - The end port (or ICMP code if protocol is "icmp")
 * `protocol` – The protocol used
 * `description` – Description of the rule
+
+## Import
+
+Individual ingress rules can be imported using `SGID_IN_PROTOCOL_PORT_SOURCE[_SOURCE]*`. Egress rules are imported similarly using `SGID_OUT_PROTOCOL_PORT_DESTINATION[_DESTINATION]*`. All parts are required.
```

Comment:
I'd suggest potential updates to this format, depending on the above, e.g. `SECURITYGROUPID_TYPE_PROTOCOL_FROMPORT_TOPORT_SOURCE[_SOURCE]*` 😄 

It may also be better to spell this out instead and rely on many examples below, e.g.

```
Security Group Rules can be imported using their `security_group_id`, `type`, `protocol`, `from_port`, `to_port`, and source(s) (e.g. `cidr_block`) separated by underscores (`_`).
```

Reply:
Updated throughout.